src/website/views/components/data-table/_examples.pug

```diff
@@ -0,0 +1,2712 @@
+h2 Examples
+p.-text To render data table, use the class <code>chi-data-table</code>.
+
+h3 Base
+.example.-mb--3
+  .-p--3
+    .chi-data-table
+      .chi-data-table__head
+        .chi-data-table__row
+          .chi-data-table__cell
+            span Name
+          .chi-data-table__cell
+            span ID
+          .chi-data-table__cell
+            span Last Login
+      .chi-data-table__body
+        .chi-data-table__row
+          .chi-data-table__cell(data-label="Name")
+            div Name 1
+          .chi-data-table__cell(data-label="ID")
+            div name-1
+          .chi-data-table__cell(data-label="Last Login")
+            div Dec 18, 2020 3:26 PM
+        .chi-data-table__row
+          .chi-data-table__cell(data-label="Name")
+            div Name 2
+          .chi-data-table__cell(data-label="ID")
+            div name-2
+          .chi-data-table__cell(data-label="Last Login")
+            div Dec 18, 2020 2:38 AM
+        .chi-data-table__row
+          .chi-data-table__cell(data-label="Name")
+            div Name 3
+          .chi-data-table__cell(data-label="ID")
+            div name-3
+          .chi-data-table__cell(data-label="Last Login")
+            div Nov 5, 2020 10:15 AM
+      .chi-data-table__footer
+        nav.chi-pagination(role='navigation', aria-label='Pagination')
+          .chi-pagination__content
+            .chi-pagination__start
+              .chi-pagination__results
+                span.chi-pagination__label 240 results
+              .chi-pagination__page-size
+                .chi-dropdown
+                  button.chi-button.-flat.-px--1.chi-dropdown__trigger#pagesize-3 20
+                  .chi-dropdown__menu.-w--xs
+                    a.chi-dropdown__menu-item.-active(href='#') 20
+                    a.chi-dropdown__menu-item(href='#') 40
+                    a.chi-dropdown__menu-item(href='#') 60
+                    a.chi-dropdown__menu-item(href='#') 80
+                    a.chi-dropdown__menu-item(href='#') All
+                span.chi-pagination__label per page
+            .chi-pagination__center
```

Comment:
Can we change from double truncation to single truncation for all data table examples as we're so tight on screen real estate? The alternative would be to enlarge the grid container for this page only. Something along the lines of

```
@media only screen and (min-width: 1200px) {
  .chi .docs-container .docs-body .docs-article .chi-grid__container {
    max-width: 1400px;
  }
}
```